app/main.py

```diff
@@ -1,2 +1,46 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = 8, length_table: int = 0) -> None:
```

Comment:
remove unnecessary arguments from __init__ - user shouldn't be able to create exp of a dict with capacity and lenght_table more then initial
![image](https://user-images.githubusercontent.com/103201010/209557589-87036a9c-5fa8-4dac-bae7-66ae35243e74.png)